.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,113 @@
+name: 🐛 Bug Report
+description: Open an issue about a bug that needs fixing.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+- type: checkboxes
+  attributes:
+    label: Prerequisites
+    options:
+    - label: I have written a descriptive issue title.
+      required: true
+    - label: I have searched all issues/PRs to ensure it has not already been reported or fixed.
+      required: true
+    - label: I have verified that I am using the latest version of Scoop and corresponding bucket.
+      required: true
+- type: input
+  attributes:
+    label: Package Name
+    description: Name of package (install name) which has bug(s)
+    placeholder: e.g. 7zip (not '7-Zip')
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected/Current Behaviour
+    description: A clear and concise description of what you expected to happen and what actually happen.
+    placeholder: I am experiencing a problem with X. I think Y should be happening but Z is actually happening.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps to Reproduce
+    description: List of steps, sample code or failing test that reproduces the behavior.
+    render: console
+    placeholder: |
+      PS> scoop install tests/meson
+      Installing 'meson' (0.61.1) [64bit]
+      Loading meson-0.61.1-64.msi from cache.
+      Checking hash of meson-0.61.1-64.msi ... ok.
+      Extracting meson-0.61.1-64.msi ... done.
+      Running installer script...
+      Linking D:\Scoop\apps\meson\current => D:\Scoop\apps\meson\0.61.1
+      Creating shim for 'meson'.
+      Can't shim 'meson.exe': File doesn't exist.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Possible Solution
+    description: Do you have some suggestions on a fix for the bug?
+    placeholder: I am experiencing a problem with X. I think Y should be happening but Z is actually happening.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Scoop and Buckets Version
+    description: Paste verbatim output from `scoop --version` below.
+    render: console
+    placeholder: |
+      PS> scoop --version
+      Current Scoop version:
+      56020838 (HEAD -> master, tag: 2021-12-26, origin/master, origin/HEAD) fix(schema): Add 'license' definition (#4596)
+
+      'extras' bucket:
+      297854268 (HEAD -> master, origin/master, origin/HEAD) notepadplusplus: Update to version 8.2.1
+
+      'main' bucket:
+      391dd8d45 (HEAD -> master, origin/master, origin/HEAD) tailwindcss: Update to version 3.0.16
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Scoop Config
+    description: Paste verbatim output from `Get-Content ~\.config\scoop\config.json | ConvertFrom-Json` below.
+    render: console
+    placeholder: |
+      PS> Get-Content ~\.config\scoop\config.json | ConvertFrom-Json
```

```suggestion
    description: Paste verbatim output from `scoop config` below.
    render: console
    placeholder: |
      PS> scoop config
```

Sorry for noticing this late, but we can just run `scoop config` to get the config values nicely printed out

---

Updated the template.